### ISSUE TITLE
[suggestion] add custom class and custom css to Shortcodes for Page Builder

### ISSUE THREAD

**Issue by gianlucaponti** [ThemeFuse/Unyson#808]
I think that could be useful to add a custom class and a custom css field to Shortcodes in Page Builder adding more flexibility in front-end customization.

**Comment by moldcraft** [comment]
Have you tried [this solution](http://manual.unyson.io/en/latest/extension/shortcodes/index.html#enqueue-shortcode-dynamic-css-in-page-head)?

**Comment by gianlucaponti** [comment]
Thank for you reply moldcraft!
I started using Unyson since a very little time and maybe I make some mistakes sometimes.
Right now I have a 'framework-customizations' folder in my theme where I have duplicated the shortcode (Section) I would like to modify and this solution is working fine.

I was wondering if adding custom class and css to any shortcode element could be an interesting option available to frontend styling.

Because of my little experience with this great framework, I actually don't know if I have to duplicate the entire shortcode folder, or I can only add the modified file I need (i.e. options.php and view.php).
If it's required to duplicate the entire folder and I add the 2 additional field to many shortcodes (like buttons, section, etc.) may be it will be more complicated to maintain on updating.

Thank you again for your support!
